### ISSUE TITLE
open file will cause crash

### ISSUE THREAD

**Issue by bb33bb** [Vector35/binaryninja-api#5255]
**Version and Platform (required):**
- Binary Ninja Version:4.1.5082-dev
- OS: manjaro
- OS Version: latest
- CPU Architecture: x64

**Bug Description:**
open a file will cause crash
as following 
```
pwndbg> disassemble
Dump of assembler code for function _ZN13SidebarWidget17setRefreshQuiesceEb:
   0x00007f5b861aa260 <+0>:	endbr64
   0x00007f5b861aa264 <+4>:	push   rbx
   0x00007f5b861aa265 <+5>:	mov    rbx,rdi
=> 0x00007f5b861aa268 <+8>:	mov    rdi,QWORD PTR [rdi+0x1b8]
   0x00007f5b861aa26f <+15>:	test   rdi,rdi
   0x00007f5b861aa272 <+18>:	je     0x7f5b861aa289 <_ZN13SidebarWidget17setRefreshQuiesceEb+41>
   0x00007f5b861aa274 <+20>:	test   sil,sil
   0x00007f5b861aa277 <+23>:	je     0x7f5b861aa280 <_ZN13SidebarWidget17setRefreshQuiesceEb+32>
   0x00007f5b861aa279 <+25>:	pop    rbx
   0x00007f5b861aa27a <+26>:	jmp    0x7f5b85dd4ca0 <_ZN6QTimer4stopEv@plt>
   0x00007f5b861aa27f <+31>:	nop
   0x00007f5b861aa280 <+32>:	call   0x7f5b85dd14a0 <_ZNK6QTimer8isActiveEv@plt>
   0x00007f5b861aa285 <+37>:	test   al,al
   0x00007f5b861aa287 <+39>:	je     0x7f5b861aa290 <_ZN13SidebarWidget17setRefreshQuiesceEb+48>
   0x00007f5b861aa289 <+41>:	pop    rbx
   0x00007f5b861aa28a <+42>:	ret
   0x00007f5b861aa28b <+43>:	nop    DWORD PTR [rax+rax*1+0x0]
   0x00007f5b861aa290 <+48>:	mov    rdi,QWORD PTR [rbx+0x1b8]
   0x00007f5b861aa297 <+55>:	pop    rbx
   0x00007f5b861aa298 <+56>:	jmp    0x7f5b85dd5480 <_ZN6QTimer5startEv@plt>
End of assembler dump.
pwndbg> info reg rdi
rdi            0x0                 0
```

**Steps To Reproduce:**
Please provide all steps required to reproduce the behavior:
1-go to open file from history list
2-crash
**Expected Behavior:**
no crash

**Screenshots:**
video has updated to slack
kindly tell me to update it here

**Additional Information:**
no

**Comment by bb33bb** [comment]
[krwx.ko.tar.gz](https://github.com/Vector35/binaryninja-api/files/14927168/krwx.ko.tar.gz)

**Comment by bb33bb** [comment]
drag it to binaryNinja also can crash the latest version .

**Comment by bb33bb** [comment]
https://github.com/Vector35/binaryninja-api/assets/5463104/1aad56e5-f92e-4a38-8fb9-0123f6df2718

**Comment by bpotchik** [comment]
Fixed in `4.1.5083-dev`.
